src/main/kotlin/org/wycliffeassociates/tstudio2rc/Converter.kt

```diff
@@ -0,0 +1,149 @@
+package org.wycliffeassociates.tstudio2rc
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.wycliffeassociates.resourcecontainer.entity.Manifest
+import org.wycliffeassociates.tstudio2rc.TextToUSFM.Companion.getVersification
+import java.io.File
+import java.nio.file.Files
+import java.util.zip.ZipFile
+
+/**
+ * API for converting a tstudio project to resource container(s).
+ */
+class Converter {
+
+    private val verseCounts = getVersification()
+
+    fun convertToRC(inputFile: File, outputDir: File): File {
+        val tempDir = Files.createTempDirectory("tempDir").toFile()
```

Comment:
can we inject this instead? I like that Orature provides its own temp dir so we can clean it up if there was an exception thrown

Reply:
I could create a tempDir inside the output dir